tests/testthat/test-gcalendr.R

```diff
@@ -1,64 +1,57 @@
-context("Read calendar events")
```

Comment:
`context()`s are no longer needed and are semi-discouraged. I left them in the other test files but got rid of this one.